sycl/doc/extensions/supported/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md

```diff
@@ -1072,3 +1100,35 @@ int main(void) {
 ```
 more examples can be found in the
 [ESIMD test suite](https://github.com/intel/llvm/tree/sycl/sycl/test-e2e/ESIMD) on github.
+
+## Implementation restrictions
+
+This section contains lists of the main restrictions that apply when using the ESIMD
+extension.
+> **Note**: Some restrictions are not enforced by the compiler, which may lead to
+> undefined program behavior if violated.
+
+### Features not supported with the ESIMD extension:
+- The [C and C++ Standard libraries support](../supported/C-CXX-StandardLibrary.rst)
+- The [Device library extensions](../../../design/DeviceLibExtensions.rst)
+
+### Unsupported standard SYCL APIs:
+
+Current ESIMD implementation does not support using certain standard SYCL features
+inside explicit SIMD kernels and functions. Most of them will be eventually
+dropped. What's not supported today:
+- Interoperability between regular SYCL and ESIMD kernels is only supported one way.
+  Regular SYCL kernels can call ESIMD functions, but not vice-versa. Invocation of SYCL code from ESIMD is not supported yet.
+- 2D and 3D target::device accessor and local_accessor;
+- Constant accessors;
+- `sycl::accessor::get_pointer()` and `sycl::accessor::operator[]` are supported only with `-fsycl-esimd-force-stateless-mem`. Otherwise, All memory accesses through an accessor are
+done via explicit APIs; e.g. `sycl::ext::intel::esimd::block_store(acc, offset)`
+- Accessors with non-zero offsets to accessed buffer;
+- Accessors with access/memory range specified;
+- `sycl::image`, `sycl::sampler` and `sycl::stream` classes.
+
+### Other restrictions:
+
+- Only Intel GPU device is supported.
+- Interoperability between regular SYCL and ESIMD kernels is only supported one way.
```

Comment:
This bullet has already been mentioned above.

Reply:
Fixed in d6b2ac049189639af1f1a13eceaeb2a38b1235e4